recipes/spiceql/README.md

```diff
@@ -0,0 +1,4 @@
+Must specify channels for conda-build to recognize packages:
```

Comment:
Please remove this file. The recipe should be building solely using packages from conda-forge.